pkg/cmd/cli/backup/describe.go

```diff
@@ -85,6 +86,7 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 
 	c.Flags().StringVarP(&listOptions.LabelSelector, "selector", "l", listOptions.LabelSelector, "only show items matching this label selector")
 	c.Flags().BoolVar(&details, "details", details, "display additional detail in the command output")
+	c.Flags().BoolVar(&insecureSkipVerify, "insecureskipverify", insecureSkipVerify, "do not verify the TLS certificate for storage requests. This is susceptible to man-in-the-middle attacks.")
```

Comment:
We should also probably clarify here that this flag is for storage bucket requests and not requests to the Kubernetes API server

Reply:
hmm... it already says `do not verify the TLS certificate for storage requests`

so I only to the end of that: `do not verify the TLS certificate for storage requests only`

Reply:
sorry I misread that, you're right that it was good as it was, but the change doesn't hurt, thanks!